server/{{appName}}-app/build.gradle.kts

```diff
@@ -12,9 +12,3 @@ sourceSets {
         }
     }
 }
-
-tasks {
-    copyDependencies {
```

Comment:
We need this task enabled as the Genesis distribution plugin relies on it to populate the distribution file's libs folder